llvm/include/llvm/Analysis/AliasAnalysis.h

```diff
@@ -635,7 +633,7 @@ class BatchAAResults {
     return AA.alias(LocA, LocB, AAQI);
   }
   bool pointsToConstantMemory(const MemoryLocation &Loc, bool OrLocal = false) {
-    return AA.pointsToConstantMemory(Loc, AAQI, OrLocal);
+    return AA.pointsToConstantMemory(Loc, OrLocal);
```

Comment:
This should call getModRefInfoMask with AAQI instead.

Reply:
Ack,